airflow/utils/log/file_task_handler.py

```diff
@@ -303,7 +303,6 @@ def _read(
         worker_log_rel_path = self._render_filename(ti, try_number)
         messages_list: list[str] = []
         remote_logs: list[str] = []
```

Comment:
I guess we could also remove this line. The IDE says it's an unused variable. (Perhaps, due to line 311)

Reply:
IDE not smart enough

doesn't understand `with suppress(NotImplementedError):`